daemon-scheduler/pkg/deployment/test_constants.go

```diff
@@ -20,7 +20,8 @@ const (
 	taskDefinition   = "arn:aws:ecs:us-east-1:12345678912:task-definition/test"
 	taskARN1         = "arn:aws:ecs:us-east-1:12345678912:task/c024d145-093b-499a-9b14-5baf273f5835"
 	taskARN2         = "arn:aws:ecs:us-east-1:12345678912:task/a1d71628-01e3-4013-b18c-6e14032a9522"
-	cluster          = "arn:aws:ecs:us-east-1:123456789123:cluster/test"
```

Comment:
why not just leave this in here and add cluster1 and cluster2? no big deal though